roles/configuration/tasks/git.yml

```diff
@@ -9,6 +9,11 @@
     configuration_git_repository_url: "{{ configuration_git_protocol }}://{{ configuration_git_host }}:{{ configuration_git_port }}/{{ configuration_git_repository }}"  # noqa 204
   when: configuration_git_username is not defined or not configuration_git_username|length
 
+- name: git - set/overwrite configuration_git_repository_url fact (with personal access token)
+  set_fact:
+    configuration_git_repository_url: "{{ configuration_git_protocol }}://{{ configuration_git_personal_access_token }}:@{{ configuration_git_host }}:{{ configuration_git_port }}/{{ configuration_git_repository }}"  # noqa 204
```

Comment:
Did you actually test this? Afaict PAT still needs to be combined with a username? Also this might leak the token if the URL is used later.

Reply:
According to https://www.edvpfau.de/kurztipp-git-clone-mit-pat-personal-access-token/ it is ``xxxx:``.